### PR TITLE
gh: Add version 0.6.1

### PR DESCRIPTION
**Pull request by h404bi**

### CODE REVIEW

File: bucket/gh.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        "64bit": {
+            "url": "https://github.com/cli/cli/releases/download/v0.5.7/gh_0.5.7_windows_amd64.zip",
+            "hash": "3b3fb943d9e902d9682ab092d35b95be468f56022474b82582bf1bcb8e6ff053"
+        },
+        "32bit": {
+            "url": "https://github.com/cli/cli/releases/download/v0.5.7/gh_0.5.7_windows_386.zip",
+            "hash": "911fa28d496e1c211478b49d6c26b0f3315887c38910dc24c3eff154150b9688"
+        }
+    },
+    "bin": "bin/gh.exe",
```
Comment (Ash258) [inline bucket/gh.json]:
```suggestion
    "bin": "bin\\gh.exe",
```

File: bucket/gh.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/cli/cli/releases/download/v$version/gh_$version_windows_amd64.zip"
+            },
+            "32bit": {
+                "url": "https://github.com/cli/cli/releases/download/v$version/gh_$version_windows_386.zip"
+            }
+        },
+        "hash": {
+            "url": "https://github.com/cli/cli/releases/download/v$version/gh_$version_checksums.txt"
```
Comment (Ash258) [inline bucket/gh.json]:
```suggestion
            "url": "$baseurl/gh_$version_checksums.txt"
```

File: bucket/gh.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
+{
+    "version": "0.5.7",
+    "homepage": "https://github.com/cli/cli",
```
Comment (Ash258) [inline bucket/gh.json]:
```suggestion
    "homepage": "https://cli.github.com",
```

File: bucket/gh.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    "description": "GitHub CLI",
```
Comment (Ash258) [inline bucket/gh.json]:
```suggestion
    "description": "Official GitHub CLI",
```

File: bucket/gh.json [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+            "url": "https://github.com/cli/cli/releases/download/v0.5.7/gh_0.5.7_windows_amd64.zip",
+            "hash": "3b3fb943d9e902d9682ab092d35b95be468f56022474b82582bf1bcb8e6ff053"
+        },
+        "32bit": {
+            "url": "https://github.com/cli/cli/releases/download/v0.5.7/gh_0.5.7_windows_386.zip",
+            "hash": "911fa28d496e1c211478b49d6c26b0f3315887c38910dc24c3eff154150b9688"
+        }
+    },
+    "bin": "bin/gh.exe",
+    "checkver": "github",
```
Comment (Ash258) [inline bucket/gh.json]:
```suggestion
    "checkver": {
        "github": "https://github.com/cli/cli"
    },
```

**Comment by equinox** [comment]
Why are we just duplicating [their own bucket](https://github.com/cli/scoop-gh)? This seems a bit redundant to me.

**Comment by Ash258** [comment]
Why would someone need to add third party bucket without automatic updates and only with one manifest??

**Comment by h404bi** [comment]
> Why are we just duplicating [their own bucket](https://github.com/cli/scoop-gh)? This seems a bit redundant to me.

Ah interesting, I didn't notice that bucket.

Pros: manifests in this main bucket can be automatically updated by the Excavator, and users don't need to add another bucket by themselves.

I would like to file an issue in that bucket repo to tell it's maintainer that the `gh` manifest can be added into Scoop's main bucket.

**Comment by equinox** [comment]
> I would like to file an issue in that bucket repo to tell it's maintainer that the gh manifest can be added into Scoop's main bucket.

Sounds like a great idea! Another pro would be scoop-related issues would end up here, rather than there where they're probably not in a position to help users as effectively as people here.

I wonder if they chose to roll their own bucket because gh's was/still is in beta (https://github.com/cli/cli/issues/280#issuecomment-580396184) rather than wanting to keep an official bucket. In which case once they find out it could be added to the main bucket I'm guessing they'll go for it

**Comment by h404bi** [comment]
Let's wait for gh developers' feedback.

**Comment by JanPokorny** [comment]
It might be possible to use an install script to automatically add `gh completion -s powershell` to PowerShell profile, similarly to Pshazz

**Comment by JanPokorny** [comment]
@Ash258 It looks like the tool they use, goreleaser, has "scoop support" which works like this: https://goreleaser.com/scoop/

File: bucket/gh.json [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/cli/cli/releases/download/v$version/gh_$version_windows_amd64.zip"
+            },
+            "32bit": {
+                "url": "https://github.com/cli/cli/releases/download/v$version/gh_$version_windows_386.zip"
+            }
+        },
+        "hash": {
+            "url": "$baseurl/gh_$version_checksums.txt"
```
Comment (mislav) [inline bucket/gh.json]: Pretty cool auto-update mechanism! 🤩

**Pull request merged**